src/generic/course-card/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { getConfig } from '@edx/frontend-platform';
+
+export const getFullImageUrl = (path: string | undefined) => `${getConfig().LMS_BASE_URL}${path}`;
```

Comment:
[question] Should the `path` parameter be an `undefined`?

Reply:
Yes, since we don't have a image of the organization yet, we need to leave it at that. I left TODO.